test/unit/deadline_submitter_for_blender/test_submitter.py

```diff
@@ -6,6 +6,12 @@
 from pathlib import Path
 
 import pytest
+from unittest.mock import Mock, patch
+
+from .mock_bpy import bpy_module as bpy
```

Comment:
do we need this mock_bpy (or that file in general) if we're mocking out the module in the `__init__.py`?

Reply:
You're right, I'd copied this structure from Houdini but the tests run fine with `bpy` mocked out in `__init__.py`. That'll be in my next revision